### ISSUE TITLE
Chained object destructuring assignment doesn't work

### ISSUE THREAD

**Issue by webbedspace** [6to5/6to5#589]
The following code isn't yet supported:
```js
({a,b}) = {c,d} = {a:1,b:2,c:3,d:4};
```
It compiles to 
```js
var _ref = { c: c, d: d } = { a: 1, b: 2, c: 3, d: 4 };
a = _ref.a;
b = _ref.b;
```
which doesn't work without native destructuring.

**Comment by sebmck** [comment]
Looks like a bug with the recent traversal rewrites. It doesn't traverse over multiple node replacements. I'll work out a fix shortly.

**Comment by sebmck** [comment]
Fixed in the 3.0.0 branch which is due out in a couple of days. Thanks!

**Comment by sebmck** [comment]
Fixed as of 3.0.0
